packages/backend/src/ee/controllers/scimOrganizationAccessTokenController.ts

```diff
@@ -32,7 +32,7 @@ import {
 import { BaseController } from '../../controllers/baseController';
 import { ServiceAccountService } from '../services/ServiceAccountService/ServiceAccountService';
 
-const SCIM_SCOPES = [ServiceAccountScope.SCIM_MANAGE];
+const SCIM_SCOPES = [ScimScope.MANAGE];
```

Comment:
why does SCIM_SCOPES flow from here into the service endpoints? Is it something you would expect eventually to come from a middleware or be on the request?

Reply:
It's because of how we structured service accounts. SCIM access is effectively a service account. The old SCIM table was renamed to service_accounts. We could do some type gymnastics to satisfy tsc 🤔 Maybe SCIM should be entirely separate from service accounts.